packages/devtools/lib/src/timeline/frame_flame_chart.dart

```diff
@@ -561,11 +562,7 @@ class TimelineGridItem extends CoreElement {
     // Update [gridLine] position.
     gridLine.element.style.left = '${width - gridLineWidth}px';
 
-    // Update [timestampLabel] position and text.
-    timestampLabel.element.style
-      ..left = '${timestampPadding}px'
-      ..width = '${width - 2 * timestampPadding}px';
-    timestampLabel.text =
-        msAsText(timestamp, fractionDigits: timestamp == 0 ? 1 : 3);
+    // Update [timestampLabel] position.
+    timestampLabel.element.style.width = '${width - 2 * timestampPadding}px';
```

Comment:
you can remove the padding from the width calculation now that you are seeing it from CSS. Essentially subtracting 2 times the padding was simulating the css box model which is now handled for you.

Reply:
discussed offline. leaving as is for now.